modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -4,30 +4,28 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-installing-aws-load-balancer-operator_{context}"]
-= Installing the AWS Load Balancer Operator
-
-You can install the AWS Load Balancer Operator from the OperatorHub by using the {product-title} web console.
+= Installing AWS Load Balancer Operator by using the web console
 
 .Prerequisites
 
 * You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
 
 * Your cluster is configured with AWS as the platform type and cloud provider.
 
+* If you are using a security token service (STS) or user-provisioned infrastructure, follow the related preparation steps. For example, if you are using AWS Security Token Service, see "Preparing for the AWS Load Balancer Operator on a cluster using the AWS Security Token Service (STS)".
```

Comment:
@alebedev87 Moved this STS info to the prerequisite section to be more in-line with modularization.
If that's okay,
1. do we have a suggested UPI doc?
2.  should this also be in the CLI procedure's prerequisites then?